lib/spring/test/acceptance_test.rb

```diff
@@ -83,14 +83,14 @@ def without_gem(name)
       end
 
       test "help message when called without arguments" do
-        assert_success "bin/spring", stdout: 'Usage: spring COMMAND [ARGS]'
+        assert_success "bundle exec bin/spring", stdout: 'Usage: spring COMMAND [ARGS]'
```

Comment:
I'm confused now :D 
Do you think this is better or with_clean_env is better ?
I think either would work, I was just sad to see bundle exec being added all over the place + that it uses a pattern we try to discourage ...